django_prometheus/db/backends/postgis/base.py

```diff
@@ -0,0 +1,44 @@
+import django
+from django.db.backends.base.base import NO_DB_ALIAS
+from django.db.backends.postgresql.base import (
+    DatabaseWrapper as Psycopg2DatabaseWrapper,
+)
+import psycopg2.extensions
+
+from django.contrib.gis.db.backends.postgis.features import DatabaseFeatures
+from django.contrib.gis.db.backends.postgis.introspection import (
+    PostGISIntrospection
+)
+from django.contrib.gis.db.backends.postgis.operations import PostGISOperations
+from django.contrib.gis.db.backends.postgis.schema import PostGISSchemaEditor
+
+from django_prometheus.db.common import DatabaseWrapperMixin, \
+    ExportingCursorWrapper
+
+
+class DatabaseWrapper(DatabaseWrapperMixin, Psycopg2DatabaseWrapper):
+    SchemaEditorClass = PostGISSchemaEditor
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if kwargs.get('alias', '') != NO_DB_ALIAS:
+            self.features = DatabaseFeatures(self)
+            self.ops = PostGISOperations(self)
+            self.introspection = PostGISIntrospection(self)
+
+    def get_connection_params(self):
+        conn_params = super(DatabaseWrapper, self).get_connection_params()
+        conn_params['cursor_factory'] = ExportingCursorWrapper(
+            psycopg2.extensions.cursor,
+            self.alias,
+            self.vendor,
+        )
+        return conn_params
+
+    def create_cursor(self, name=None):
+        # cursor_factory is a kwarg to connect() so restore create_cursor()'s
+        # default behavior
+        if django.VERSION >= (1, 11, 0):
```

Comment:
No need to support old django versions.
https://www.djangoproject.com/download/#supported-versions

Reply:
Got it.